src/main/java/io/xstefank/wildfly/bot/config/WildFlyBotConfig.java

```diff
@@ -16,4 +16,7 @@ public interface WildFlyBotConfig {
     default boolean isDryRun() {
         return dryRun().orElse(false);
     }
+
+    @WithDefault("30")
+    int timeout();
```

Comment:
be more specific about what is this timeout used for, maybe pullrequest.mergable.timeout